presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -601,6 +604,60 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
         return getTableMetadata(session, ((HiveTableHandle) tableHandle).getSchemaTableName());
     }
 
+    @VisibleForTesting
+    static Function<HiveColumnHandle, ColumnMetadata> columnMetadataGetter(Table table, TypeManager typeManager, ColumnConverter columnConverter)
```

Comment:
There is another unused method of this in this class. I guess it's an accidental copy out of the other method? Maybe we just modify the original method?